components/brave_vpn/browser/connection/BUILD.gn

```diff
@@ -23,6 +23,17 @@ source_set("api") {
     "//components/prefs",
     "//services/network/public/cpp",
   ]
+
+  if (is_win) {
+    sources += [
+      "brave_vpn_os_connection_api_win.cc",
+      "brave_vpn_os_connection_api_win.h",
+    ]
+    deps += [
+      "//chrome/elevation_service:public_headers",
```

Comment:
see above comment on DEPS, these are laying violations

Reply:
Fixed in https://github.com/brave/brave-core/commit/6f77a96a2c5e41feedf1946827c5b97f3429233d